build_product_images.py

```diff
@@ -39,6 +49,10 @@ def parse_args():
     )
     parser.add_argument("-u", "--push", help="Push images", action="store_true")
     parser.add_argument("-d", "--dry", help="Dry run.", action="store_true")
+    parser.add_argument("-a", "--architecture", help="Target platform for image")
+    parser.add_argument("-c", "--check", help="Setting the flag will enable dependency checks and building layers", action="store_true")
+    parser.add_argument("-m", "--multiarch", help="Build and publish multi-architecture images", action="store_true")
```

Comment:
This is not needed. I can pass multiple architectures with the `-a` argument.

Reply:
Done